package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "calcite-app-components",
+  "name": "@esri/calcite-app-components",
   "version": "2.0.2",
-  "description": "A collection of calcite components for building single page applications.",
+  "description": "A collection of web components for building single-page mapping applications.",
```

Comment:
I don't think we need to say `mapping`. it will work with anything in the center and not require a map.

Reply:
k removed.